exchange/auction_test.go

```diff
@@ -245,13 +263,13 @@ func runCacheSpec(t *testing.T, fileDisplayName string, specData *cacheSpec) {
 		winningBidsByBidder: winningBidsByBidder,
 		roundedPrices:       roundedPrices,
 	}
-	_ = testAuction.doCache(ctx, cache, targData, &specData.BidRequest, 60, &specData.DefaultTTLs, bidCategory, &specData.DebugLog)
+	_ = testAuction.doCache(ctx, cache, targData, &specData.BidRequest, 60, &specData.DefaultTTLs, bidCategory, &specData.DebugLog, isVideo)
 
 	if len(specData.ExpectedCacheables) > len(cache.items) {
 		t.Errorf("%s:  [CACHE_ERROR] Less elements were cached than expected \n", fileDisplayName)
-	} else if len(specData.ExpectedCacheables) < len(cache.items) {
+	} else if len(specData.ExpectedCacheables) < len(cache.items) && !isVideo {
 		t.Errorf("%s:  [CACHE_ERROR] More elements were cached than expected \n", fileDisplayName)
-	} else { // len(specData.ExpectedCacheables) == len(cache.items)
+	} else if len(specData.ExpectedCacheables) == len(cache.items) && isVideo {
```

Comment:
I think you can just leave this as an 'else' here (leaving the condition commented out like it was before). In the event that isVideo and more bids than expected were cached it looks to me like the below code will still work (which just checks that all of the expected cachables were indeed cached, and that should still be the case).

Reply:
Right, fixed.